Realm/ObjectServerTests/run_baas.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/sh
+
+######################################
+#
+# Copyright 2020 Realm Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+######################################
+
+mongodb_url=http://fastdl.mongodb.org/osx/mongodb-osx-ssl-x86_64-4.0.2.tgz;
+transpiler_target=node8-macos;
+server_stitch_lib_url="https://s3.amazonaws.com/mciuploads/mongodb-mongo-master/stitch-support/osx-1010/80f9a13324fc36b2deb400e5a185968f6fa8f64a/stitch-support-4.1.7-319-g80f9a13324.tgz";
```

Comment:
Pulling all of the various version numbers that appear in the script below up to here would make updating things easier.